src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -85,7 +85,7 @@ ContextAttribute::ContextAttribute(ContextAttribute* caP)
   numberValue           = caP->numberValue;
   boolValue             = caP->boolValue;
   compoundValueP        = caP->compoundValueP;
-  caP->compoundValueP   = NULL;
+  caP->compoundValueP   = NULL;   // FIXME P10: really?? shouldn't it be a clone?
```

Comment:
I think is ok, as long as the behaviour is clear and and the code (taking into account the changes in this PR) works. I'll add a comment in the constructor preamble to explain it.

(Btw, note that compound are not cloned, but metadata aren't... This different approach in two different "children" ContextAttribute is also a bit confusing. I'll hope to clarify also this in the comment).